HelpAssistHer/help-assist-her
# Header scaffolding

## Description
We have a candidate who is interviewing to volunteer and her task is going to be to complete the success header message, shown in the designs here: https://projects.invisionapp.com/d/main#/console/11323423/296756514/preview

This PR removes the old header we had that wasn't doing anything, moves the Login button down closer to where it should be, and adds a component file (scaffolding) so the candidate can complete her task.

## Test Plan
1. Go to the verification portal and see that it looks like this screenshot. The space at the top is where the header success message will go. Note: this component will only appear once the Save button is clicked, but for right now, I'm having her create it as a static component (baby steps 👶 ).
<img width="1436" alt="screen shot 2018-06-06 at 1 05 36 pm" src="https://user-images.githubusercontent.com/6378435/41062339-649437c4-698a-11e8-9206-c0ee49e5380c.png">

2. Make sure the Login button still works
